ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_arithmetic_operations.py

```diff
@@ -434,3 +434,57 @@ def test_numpy_floor_divide(
         subok=True,
         test_values=False,
     )
+
+
```

Comment:
```suggestion

# mod
```

Reply:
I think we can just add this small comment for consistency

Reply:
Comment Added